addons/hide-project-stats/userscript.js

```diff
@@ -0,0 +1,102 @@
+export default async function ({ addon, global, console }) {
+  let loves = { name: "love" };
+  let favorites = { name: "favorite" };
+
+  /**
+   * Checks whether or not the currently signed-in user matches the project owner.
+   * @returns A boolean value.
+   */
+  async function isVisitingOwnProject() {
+    let username = await addon.auth.fetchUsername();
+    await addon.tab.waitForElement(".title");
+    return username !== null && document.querySelector(".title form");
```

Comment:
```suggest
    let username = await addon.auth.fetchUsername();
    if (!username) return false
    return !!await addon.tab.waitForElement(".title form");
 ```

Reply:
confusing, waitForElement will never resolve to a falsey value